pom.xml

```diff
@@ -254,6 +257,70 @@
         <module>its</module>
       </modules>
     </profile>
+    <profile>
+      <id>enable-rule-api</id>
```

Comment:
Witohut a profile, something conflicts with RE scripts and the build fails somewhere in the beginning